src/discord/devtools.py

```diff
@@ -111,6 +111,17 @@ async def hello(self, interaction: discord.Interaction):
             "Well, hello there. Welcome to version 5!",
         )
 
+    @app_commands.command()
+    @app_commands.guilds(*env.slash_command_guilds)
+    @app_commands.check(is_in_bot_spam)
+    async def version(self, interaction: discord.Interaction):
+        """
+        Displays the current bot version and the current Git commit the bot is running on.
+        """
+        await interaction.response.send_message(
+            f"Version: {self.bot.__version__}\nCommit: {self.bot.__commit__ or 'Unknown'}",
+        )
+
```

Comment:
I don't think we really need another command for this, since `/about` already shows the version. Maybe just stick the commit in the footer of the embed.

Reply:
Didn't see the existing command, ty!